common/core/desktop/src/kmx/kmx_file.cpp

```diff
@@ -18,6 +19,37 @@ KMX_BOOL KMX_ProcessEvent::Load(km_kbp_path_name KeyboardName)
   return TRUE;
 }
 
+const int km::kbp::kmx::CODE__SIZE[] = {
+    -1,  // undefined                0x00
+    1,   // CODE_ANY                 0x01
+    2,   // CODE_INDEX               0x02
+    0,   // CODE_CONTEXT             0x03
+    0,   // CODE_NUL                 0x04
+    1,   // CODE_USE                 0x05
+    0,   // CODE_RETURN              0x06
+    0,   // CODE_BEEP                0x07
+    1,   // CODE_DEADKEY             0x08
+    0,   // CODE_BACKSPACE           0x09
+    2,   // CODE_EXTENDED            0x0A
+    0,   // CODE_EXTENDEDEND         0x0B
+    0,   // CODE_SWITCH              0x0C
+    0,   // CODE_KEY                 0x0D
+    1,   // CODE_CLEARCONTEXT        0x0E
+    1,   // CODE_CALL                0x0F
+    0,   // UC_SENTINEL_EXTENDEDEND  0x10
+    1,   // CODE_CONTEXTEX           0x11
+    1,   // CODE_NOTANY              0x12
+    2,   // CODE_KEYMAN70_LASTCODE   0x13
+    3,   // CODE_IFOPT               0x14
+    1,   // CODE_SAVEOPT             0x15
+    1,   // CODE_RESETOPT            0x16
+    3,   // CODE_IFSYSTEMSTORE       0x17
+    2    // CODE_LASTCODE            0x18
+};
+
+// Ensure that all CODE_### sizes are defined
+static_assert(sizeof(CODE__SIZE) / sizeof(CODE__SIZE[0]) == (CODE_LASTCODE + 1), "Size of array CODE_SIZE not correct");
```

Comment:
```suggestion
static_assert(sizeof(CODE__SIZE) / sizeof(CODE__SIZE[0]) == (CODE_LASTCODE + 1), "Size of array CODE__SIZE not correct");
```

Reply:
compared it with CODE_SETSYSTEMSTORE+1 instead of CODE_LASTCODE +1